src/current/_includes/cockroachcloud/download-the-binary.md

```diff
@@ -24,7 +32,7 @@
 
     {% include_cached copy-clipboard.html %}
     ~~~ shell
-    $ErrorActionPreference = "Stop"; [Net.ServicePointManager]::SecurityProtocol = [Net.SecurityProtocolType]::Tls12;$ProgressPreference = 'SilentlyContinue'; $null = New-Item -Type Directory -Force $env:appdata/cockroach; Invoke-WebRequest -Uri https://binaries.cockroachdb.com/cockroach-{{ page.release_info.version }}.windows-6.2-amd64.zip -OutFile cockroach.zip; Expand-Archive -Force -Path cockroach.zip; Copy-Item -Force "cockroach/cockroach-{{ page.release_info.version }}.windows-6.2-amd64/cockroach.exe" -Destination $env:appdata/cockroach; $Env:PATH += ";$env:appdata/cockroach"
+    $ErrorActionPreference = "Stop"; [Net.ServicePointManager]::SecurityProtocol = [Net.SecurityProtocolType]::Tls12;$ProgressPreference = 'SilentlyContinue'; $null = New-Item -Type Directory -Force $env:appdata/cockroach; Invoke-WebRequest -Uri https://get.cockroachdb.com/dl/cockroach-latest.windows-6.2-amd64.zip -OutFile cockroach.zip; Expand-Archive -Force -Path cockroach.zip; Copy-Item -Force "cockroach/cockroach-latest.windows-6.2-amd64/cockroach.exe" -Destination $env:appdata/cockroach; $Env:PATH += ";$env:appdata/cockroach"
```

Comment:
(same)
instead of: `cockroach/cockroach-latest.windows-6.2-amd64/cockroach.exe`
should be: `cockroach/cockroach-*/cockroach.exe`

![image](https://github.com/cockroachdb/docs/assets/3051672/702b3354-1684-4f49-a024-333dc585a112)

Reply:
Worked around this by relying on the Connect modal.